src/components/hooks/useMeeting.js

```diff
@@ -1,28 +1,41 @@
 import {useEffect, useContext, useState} from 'react';
+import {tap} from 'rxjs/operators';
 
 import {AdapterContext} from '../../components/';
 
 /**
  * Custom hook that returns meeting data of the given ID.
+ * If no ID is given, a `meetingDestination` must be provided to
+ * create a new meeting.
  *
  * @param {string} meetingID  ID of the meeting for which to return data.
+ * @param {string} meetingDestination  Virtual location where the meeting should take place
  * @returns {Meeting} Data of the meeting
  */
-export default function useMeeting(meetingID) {
+export default function useMeeting(meetingID, meetingDestination) {
   const [meeting, setMeeting] = useState({});
   const {meetingsAdapter} = useContext(AdapterContext);
 
   useEffect(() => {
+    let subscription;
     const onError = (error) => {
       throw error;
     };
-    const subscription = meetingsAdapter.getMeeting(meetingID).subscribe(setMeeting, onError);
+
+    if (meetingID) {
+      subscription = meetingsAdapter
+        .getMeeting(meetingID)
+        .pipe(tap(({ID}) => meetingsAdapter.addLocalMedia(ID)))
```

Comment:
getMeeting must be subscribed **Before** adding the localMedia. Otherwise, we will not receive the local media.

Reply:
My bad, it's on the wrong place 🙁